setup-scripts/blocklist-airtable.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import traceback, os, asyncio, requests, json, discord
+from bot_utils import setup, send_msg
+
+bot_token = setup()
+client = discord.Client()
+
+AIRTABLE_API_KEY = os.getenv("AIRTABLE_API_KEY")
+AIRTABLE_BASE = os.getenv("AIRTABLE_BASE", "app89plJvA9EqTJEc")
+AIRTABLE_TABLE = os.getenv("AIRTABLE_TABLE", "Table%201")
+AIRTABLE_FIELD = os.getenv("AIRTABLE_FIELD", "Link")
+
+
+def exec(command):
+    return os.popen(command).read().strip()
+
+
+async def block_skylinks_from_airtable():
+    print("Pulling blocked skylinks from Airtable via api integration")
+    headers = {"Authorization": "Bearer " + AIRTABLE_API_KEY}
+    skylinks = []
+    offset = None
+    while len(skylinks) == 0 or offset:
+        print("Requesting a batch of records from Airtable with " + (offset if offset else "empty") + " offset")
+        query = "&".join(["fields%5B%5D=" + AIRTABLE_FIELD, ("offset=" + offset) if offset else ""])
+        response = requests.get(
+            "https://api.airtable.com/v0/" + AIRTABLE_BASE + "/" + AIRTABLE_TABLE + "?" + query,
+            headers=headers,
+        )
+
+        if response.status_code != 200:
+            status_code = str(response.status_code)
+            response_text = response.text or "empty response"
+            message = "Airtable blocklist integration responded with code " + status_code + ": " + response_text
+            return print(message) or await send_msg(client, message, force_notify=False)
```

Comment:
I don't get this line. If `print` returns `true` the `send_msg` will never get called. If it returns `false` it will always get called. Why the logical operator?

Reply:
print always return null so send_msg will always be invoked - the message will be printed for debugging purposes and also sent to discord

I wanted to have them in one line because they basically deal with the same message and print is just to debug the message output when running manually.

Reply:
Hm, I've never seen that pattern in situations where both logical branches will always be executed. I find it a little confusing and I would generally prefer to have them on two separate lines for clarity but it's not a big deal.